generator/Cargo.toml

```diff
@@ -1,17 +1,20 @@
 [package]
 name = "generator"
-version = "1.1.0"
+version = "1.1.1"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
 concordium-rust-sdk = { path = "../deps/concordium-rust-sdk", version = "*" }
-clap = {version = "4", features = ["derive", "color"] }
+clap = { version = "4", features = ["derive", "color"] }
 anyhow = "1"
-chrono = {version = "0.4", features = ["serde"] }
+chrono = { version = "0.4", features = ["serde"] }
 rand = "0.7"
-tokio = {version = "1.27", features = ["rt-multi-thread", "macros", "time"]}
-tonic = {version = "0.8", features = ["tls", "tls-roots"]} # Use system trust roots.
+tokio = { version = "1.27", features = ["rt-multi-thread", "macros", "time"] }
+tonic = { version = "0.8", features = [
+    "tls",
+    "tls-roots",
+] } # Use system trust roots.
```

Comment:
I said this many times already, but please don't reformat things like this. Please revert these changes and control your editor.